tracing/src/test/java/com/palantir/tracing/TracersTest.java

```diff
@@ -38,6 +38,7 @@
 import org.mockito.MockitoAnnotations;
 import org.slf4j.MDC;
 
+@SuppressWarnings("deprecation")
```

Comment:
where is the deprecation coming from here?

Reply:
Several tests for Tracers methods we have deprecated